### PR TITLE
Add support for browsers via browserify

### PR DESCRIPTION
**Pull request by voronianski**
On hold for now, until https://github.com/HarasimowiczKamil/any-base/pull/1 will be merged!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,15 +26,18 @@
   },
   "homepage": "https://github.com/oculus42/short-uuid#readme",
   "dependencies": {
-    "any-base": "^1.0.0",
+    "any-base": "voronianski/any-base",
```
Comment (voronianski) [inline package.json]: I'm using fork for now (until https://github.com/HarasimowiczKamil/any-base/pull/1 will be merged).

Comment (oculus42) [inline package.json]: any-base doesn't need to be updated for short-uuid to support browserify, so we can eliminate this change.

**Comment by oculus42** [comment]
I pulled down the changes and ran browserify locally. It produced a ~600K file (before uglify) from the original ~14K of uncompressed sources. I would like to make short-uuid available for client-side use, but that expansion is flat-out crazy.

This seems to be because node-uuid checks for node or browser and even though it doesn't need all the libraries, browserify is helpfully loading them for us. If I manually remove the Node references from node-uuid, Browserify comes down to ~13.8K. I'm going to dig in and see if we can make this smaller.

As a side-note I tried webpack. It produced a ~130K file. Better, but still crazy.

**Comment by voronianski** [comment]
@oculus42 this can be closed now?

**Comment by oculus42** [comment]
As of short-uuid@2.1.1 the dist files are in the npm package, as requested. Closing.

**Comment by oculus42** [comment]
Thank you for suggesting it.

**Pull request closed without merging**